### PR TITLE
E2E Testing Integration

### PR DESCRIPTION
**Pull request by nalancer08**
### Type of change

- [ ] Chore
- [X] Feature
- [ ] Bug fix

### Linked issue

- [X] Not tracked

### Why
Test DemoApp and include to Firebase TestLab CI and E2E testing with Android UI and Unit Test over DemoApp

### CODE REVIEW

File: app/src/androidTest/java/app/cybrid/demoapp/ui/tradeFlow/TradeFlowActivityTest.kt [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+
+        //compose. onNodeWithText("Bitcoin").assertIsDisplayed()
+        Thread.sleep(3000)
+        //compose2.onNodeWithTag("QuoteComponent").assertIsDisplayed()
+
+        //compose.onNodeWithTag("QuoteComponent").assertIsDisplayed()
+        //compose.waitForIdle()
+        //compose.onNodeWithText("Bitcoin").assertIsDisplayed()
+        //compose.onNodeWithText("BTC").assertIsDisplayed()
+        //.sleep(3000)
```
Comment (jc-cybrid) [inline app/src/androidTest/java/app/cybrid/demoapp/ui/tradeFlow/TradeFlowActivityTest.kt:73]: Any reason to leave this commented code?

File: app/src/main/java/app/cybrid/demoapp/ui/listComponents/entity/Component.kt [hunk 2]
```diff
@@ -18,11 +18,11 @@ class Component() {
 
             // -- ListPrices Component
             val pricesComponents = Component(1, "ListPrices Component")
-            components.add(pricesComponents)
+            //components.add(pricesComponents)
```
Comment (jc-cybrid) [inline app/src/main/java/app/cybrid/demoapp/ui/listComponents/entity/Component.kt:21]: Remove comment.

File: app/src/main/java/app/cybrid/demoapp/ui/listComponents/entity/Component.kt [hunk 3]
```diff
@@ -18,11 +18,11 @@ class Component() {
...
 
             // -- ListPrices Assets Component
             val pricesAssetsComponents = Component(2, "ListPrices + Assets Component")
-            components.add(pricesAssetsComponents)
+            //components.add(pricesAssetsComponents)
```
Comment (jc-cybrid) [inline app/src/main/java/app/cybrid/demoapp/ui/listComponents/entity/Component.kt:25]: Remove comment.

**Review by jc-cybrid — CHANGES_REQUESTED** [review]
Remove commented code

**Review by jc-cybrid — APPROVED** [review]

**Review by brent-cybrid — APPROVED** [review]

File: app/build.gradle [hunk 4]
```diff
@@ -65,6 +65,12 @@ dependencies {
 
     androidTestImplementation 'androidx.test.ext:junit:1.1.2'
     androidTestImplementation 'androidx.test.espresso:espresso-core:3.3.0'
+    androidTestImplementation 'androidx.test:rules:1.3.0'
+    androidTestImplementation 'androidx.test:runner:1.3.0'
+    //androidTestImplementation 'androidx.test:core-ktx:1.8.0'
```
[the remark this answers is not in the document]
Comment (nalancer08) [reply app/build.gradle:70]: Yes!

**Pull request merged**